### PR TITLE
Added minimal sanity bucket for development unit test

### PR DESCRIPTION
**Pull request by harihare**
Enable sanity bucket with minimal run time for development unit test.

### CODE REVIEW

File: config/tests/guest/libvirt/sanity_dev.cfg [hunk 1]
```diff
@@ -0,0 +1,124 @@
...
+                    - uri:
+                        only virsh.uri.no_option.connect_to_local,virsh.uri.unexpect_option
+                    - version:
+                        only virsh.version.no_option,virsh.version.unexpect_option
+                    - vncdisplay:
+                        only virsh.vncdisplay.normal_test.id_option,virsh.vncdisplay.error_test.no_option
+                    - shutdown:
+                       only virsh.shutdown.normal_test.id_option.no_mode,virsh.shutdown.error_test.no_option
+#                   - reboot:
+#                       only virsh.reboot.normal_test.id_option.no_mode,virsh.reboot.error_test.no_option
```
Comment (PraveenPenguin) [inline config/tests/guest/libvirt/sanity_dev.cfg:102]: please remove commented test case

File: config/tests/guest/libvirt/sanity_dev.cfg [hunk 2]
```diff
@@ -0,0 +1,124 @@
...
+                        only virsh.domblkerror.undefinded_error,virsh.domblkerror.nospace_error
+                    - itself:
+                        only virsh.itself.normal_test.default_option,virsh.itself.error_test.invalid_cmd
+                    - save:
+                        only virsh.save.normal_test.id_option.no_option.no_progress,virsh.save.error_test.no_option
+                    - sendkey:
+                        only virsh.sendkey.params_test.non_acl.without_codeset,virsh.sendkey.sysrq.non_acl.show_memory_usage
+                    - delete_guest:
+                        only remove_guest.without_disk
+
```
Comment (PraveenPenguin) [inline config/tests/guest/libvirt/sanity_dev.cfg:124]: whitespace please

**Review by PraveenPenguin — COMMENTED** [review]
@harihare have some style comment, can you please update run log as well

**Review by spoorthys1303 — APPROVED** [review]
LGTM

**Pull request merged**